v4-client-js/src/clients/modules/post.ts

```diff
@@ -168,28 +218,62 @@ export class Post {
      * @description Calculate the default broadcast mode.
      */
     private defaultBroadcastMode(msgs: EncodeObject[]): BroadcastMode {
-      if (
-        msgs.length === 1 &&
-        (msgs[0].typeUrl === '/dydxprotocol.clob.MsgPlaceOrder' ||
-          msgs[0].typeUrl === '/dydxprotocol.clob.MsgCancelOrder')
-      ) {
-        const orderFlags = msgs[0].typeUrl === '/dydxprotocol.clob.MsgPlaceOrder'
-          ? (msgs[0].value as MsgPlaceOrder).order?.orderId?.orderFlags
-          : (msgs[0].value as MsgCancelOrder).orderId?.orderFlags;
-
-        switch (orderFlags) {
-          case OrderFlags.SHORT_TERM:
-            return Method.BroadcastTxSync;
-
-          case OrderFlags.LONG_TERM:
-          case OrderFlags.CONDITIONAL:
-            return Method.BroadcastTxCommit;
-
-          default:
-            break;
-        }
+      return this.areShortTermOrderMsgs(msgs) ? Method.BroadcastTxSync : Method.BroadcastTxCommit;
+    }
+
+    /**
+     * @description Calculate the default broadcast mode for a msg.
+     */
+    private defaultMsgBroadcastMode(msg: EncodeObject): BroadcastMode {
+      if (this.isShortTermOrderMsg(msg)) {
+        return Method.BroadcastTxSync;
+      } else {
+        return Method.BroadcastTxCommit;
```

Comment:
can shorten to `return this.isShortTermOrderMsg(msg) ? Method.BroadcastTxSync : Method.BroadcastTxCommit;`

Reply:
similar to what we have in `defaultBroadcastMode`